tests/test_visualizer/test_vis_backend.py

```diff
@@ -285,3 +286,46 @@ def test_close(self):
         mlflow_vis_backend.add_config(cfg)
         mlflow_vis_backend.close()
         shutil.rmtree('temp_dir')
+
+
+class TestClearMLVisBackend:
```

Comment:
```suggestion
@patch.dict(sys.modules, {'clearml': MagicMock()})
class TestClearMLVisBackend:
```

Reply:
I got this error: 

tests/test_visualizer/test_vis_backend.py:291:2: F821 undefined name 'patch'

======================================================================================================================================== ERRORS ========================================================================================================================================
______________________________________________________________________________________________________________ ERROR collecting tests/test_visualizer/test_vis_backend.py ______________________________________________________________________________________________________________
tests/test_visualizer/test_vis_backend.py:291: in <module>
    @patch.dict(sys.modules, {'clearml': MagicMock()})
E   NameError: name 'patch' is not defined

Reply:
You need to import patch by `from unittest.mock import patch` before using it

Reply:
Added.